python/usepyhf/plot.py

```diff
@@ -0,0 +1,41 @@
+import pyhf
+import numpy as np
+import matplotlib.pyplot as plt
+from pyhf.contrib.viz import brazil
+from usepyhf import util
+
+def brazil_plot(workspace:pyhf.Workspace, test_stat:str="qtilde", size=[7, 5]):
+    """
+    Make a brazil band plot to show the upper limit CI and +/- [1, 2] sigma CL.
+    """
+    #Extract model and data
+    model = workspace.model()
+    data = workspace.data(model)
+
+    #Find parameter of interest
+    poi_index = model.config.poi_index
+    poi_name = model.config.par_order[poi_index]
+
+    #Find bounds
+    poi_min, poi_max = model.config.suggested_bounds()[poi_index]
```

Comment:
Don't forget the bounds here, which can use the config manager scan range if not None